_includes/pbc/all/install-features/202404.0/install-the-merchant-b2b-contracts-feature.md

```diff
@@ -249,9 +249,9 @@ class CompanyUserDependencyProvider extends SprykerCompanyUserDependencyProvider
 
 {% info_block warningBox "Verification" %}
 
-When a new merchant relationship is being established, there is a property called `CompanyBusinessUnit.merchantRelationships` in the assigned business units. 
-This property should be updated with the data of the newly created merchant relationship. 
-This update should be verified while you are logged in as a company user who is a member of the assigned business unit.
+1. Create a merchant relationship.
+2. Log in as a company user who's a member of the business unit assigned to the relationship.
+    Make sure the `CompanyBusinessUnit.merchantRelationships` property was updated with the merchant relationship data.
```

Comment:
I assume this happens on the Storefront. How do I check property related information there?

Reply:
Extended the check with more detailed steps required to do.